forms/jqm/element.js

```diff
@@ -158,6 +166,10 @@ define(function (require) {
 
     },
 
+    onChangeClass: function () {
+      this.$el.attr('class', this.model.attributes['class']);
```

Comment:
is it possible that the element might have other classes on it? user defined ones? .attr('class') overwrite all of them